src/pyload/webui/app/blueprints/cnl_blueprint.py

```diff
@@ -48,7 +51,7 @@ def add():
         "package", flask.request.form.get("source", flask.request.form.get("referer"))
     )
 
-    urls = [url for url in flask.request.form["urls"].split("\n") if url.strip()]
+    urls = [url for url in flask.request.form["urls"].replace(' ', '\n').split("\n") if url.strip()]
```

Comment:
why this change?

Reply:
@GammaC0de : I wrote you an email regarding this topic.

Reply:
Does JDownloader handle this correctly?

Reply:
Yes it does:
![grafik](https://user-images.githubusercontent.com/6438895/98450758-a3d16f00-213f-11eb-8444-82ae0804c835.png)

![grafik](https://user-images.githubusercontent.com/6438895/98450771-d11e1d00-213f-11eb-95d7-376646bc7c03.png)